sdk/storage/azure-storage-blob/tests/test_block_blob_sync_copy.py

```diff
@@ -76,16 +76,17 @@ def test_put_block_from_url_and_commit(self):
         dest_blob = self.bsc.get_blob_client(self.container_name, dest_blob_name)
 
         # Act part 1: make put block from url calls
+        split = 4 * 1024
```

Comment:
Is there a reason we want to call it split?

Reply:
hmm, because we are doing `stage_block_from_url` in line 85 from this "split" point ( 0-split, followed by split = 2*split)

Reply:
haha the name is a little bit confusing, while it's fine I think, we can rename it at any time ^_^